src/client/src/containers/Footer/styles.module.css

```diff
@@ -41,6 +41,10 @@
   color: #ffffff;
 }
 
+.nextIcon path {
+  fill: var(--vscode-editor-foreground);
```

Comment:
Would it break anything if we simply had something like this in a global css file? To prevent this issue from happening going forward.

```css
path {
    fill: var(--vscode-editor-foreground);
}
```

Reply:
Looking at it, it seems to to me a good idea. Having said that, I am not a CSS person, so not quite sure. @annielovesjs  @jsondoo Do you have any comments on this?

Reply:
SVG icons that want to have special colors other than the foreground color will still work as they will override the `path` rule. So basically we can remove all the css rules that have `fill: var(--vscode-editor-foreground);` and instead have what I wrote above. For checkmark and stuff like that it will be as it is right now

Reply:
Per discussion we will leave this one as is for now as implementing the project-wide `path` rule breaks a lot of things currently